.gitignore

```diff
@@ -24,3 +24,6 @@ htmlcov/
 .pytest_cache/
 cupy/.data/
 cupy/cuda/thrust.h
+
+# Ignore generated files
+cupy/_core/numpy_allocator.h
```

Comment:
This should be fixed by #8735, so could you exclude this commit? LGTM except that!

Reply:
Done! Didn't notice that PR.